onnxruntime/test/perftest/main.cc

```diff
@@ -41,17 +41,6 @@ int real_main(int argc, char* argv[]) {
     if (failed)
       return -1;
   }
-
-  if (test_config.machine_config.provider_type_name == onnxruntime::kOpenVINOExecutionProvider) {
-    if (test_config.run_config.concurrent_session_runs != 1) {
-      fprintf(stderr, "OpenVINO doesn't support more than 1 session running simultaneously default value of 1 will be set \n");
-      test_config.run_config.concurrent_session_runs = 1;
-    }
-    if (test_config.run_config.execution_mode == ExecutionMode::ORT_PARALLEL) {
-      fprintf(stderr, "OpenVINO doesn't support parallel executor using sequential executor\n");
-      test_config.run_config.execution_mode = ExecutionMode::ORT_SEQUENTIAL;
-    }
-  }
```

Comment:
same changes should apply to onnx_test_runer tool also [https://github.com/microsoft/onnxruntime/blob/7da5949279577ee21fa09ed6484833dfe10f127d/onnxruntime/test/onnx/main.cc#L320-L327](https://github.com/microsoft/onnxruntime/blob/7da5949279577ee21fa09ed6484833dfe10f127d/onnxruntime/test/onnx/main.cc#L320-L327)

Reply:
> same changes should apply to onnx_test_runer tool also
> 
> https://github.com/microsoft/onnxruntime/blob/7da5949279577ee21fa09ed6484833dfe10f127d/onnxruntime/test/onnx/main.cc#L320-L327

@HectorSVC  sure, will make this changes for onnx_test_runner tool as well.
After, I made those changes locally, now when I try to run:
 **./onnx_test_runner -r 2 -c 2 -e openvino** /home/maajid/project_files/data_models_onnx_and_tests/alexnet_with_1test/

It doesn't run and it throw's this message:
**when you use '-r [repeat]', please set '-c' to 1**

Is this expected behavior with onnx_test_runner tool ?
because I don't see the same with onnxruntime_perf_test.
I was able to run something like ./onnxruntime_perf_test -r 2 -c 2 -e openvino ..   successfully!!

Reply:
that's should be expected.
by the way, once we remove this, what will happen for VADMBackend?

---
In reply to: [510417855](https://github.com/microsoft/onnxruntime/pull/5568#discussion_r510417855) [](ancestors = 510417855)

Reply:
> that should be expected.
> by the way, once we remove this, what will happen for VADMBackend?
> 
> In reply to: [510417855](https://github.com/microsoft/onnxruntime/pull/5568#discussion_r510417855) [](ancestors = 510417855)

@HectorSVC As we know on an HDDL Device that uses VADMBackend, we are already considering the optimal number to be 8. we are creating 8 instances of Infer Requests when VAD-M accelerator is being used that sets number of maximum parallel inferences when using VADMBackend.

Now coming to this change where I am removing the checks in https://github.com/microsoft/onnxruntime/blob/7da5949279577ee21fa09ed6484833dfe10f127d/onnxruntime/test/onnx/main.cc#L320-L327

This won't have any impact on VADMBackend, the inference will continue to work the way it was happening earlier with VADMBackend